index.html

```diff
@@ -1944,6 +1963,109 @@ <h2>
           Visual representation of a [=Standard Gamepad=] layout.
         </figcaption>
       </figure>
+      <section>
+        <h3>
+          Extended Mapping
+        </h3>
+        <p>
+          This section introduces an extended gamepad button mapping beyond the Standard Gamepad layout.
+          These additional buttons are commonly found on certain gamepad models.
+          The following table defines the extended buttons used by some gamepad models in the following diagrams.
+          It’s important to note that this list is not exhaustive, and user agents may utilize different
+          or additional buttons for these or other gamepad models.
+          Consequently, the number of buttons on the Gamepad is not limited to the standard mapping of 17 buttons.
+        </p>
+       <p>
+          To accommodate extended gamepads, we have incorporated an "extended" mapping enumeration
+          into the GamepadMappingType. Additionally, we have defined an enumeration for the various
+          button types in the table, termed "GamepadButtonType", and have expanded the "GamepadButton"
+          interface to encompass this new button type enumeration.
+       </p>
+        <table class="tg">
+          <thead>
+            <tr>
+              <th scope="col">Buttons</th>
+              <th scope="col">Gamepad Models</th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr>
+              <td>trackpad/touchpad</td>
+              <td>Sony DualSense, DualShock 4</td>
+            </tr>
+           <tr>
+              <td>share/capture</td>
+              <td>Xbox Wireless Controller Model 1914<br>
+                Nintendo Switch controllers<br>
+                Stadia controller<br>
+              </td>
+            </tr>
+           <tr>
+              <td>voice assistant</td>
+              <td>Google Stadia Controller<br>
+                Amazon Luna Controller<br>
+              </td>
+            </tr>
+            <tr>
+              <td>home</td>
+              <td>Nvidia Shield Gamepad</td>
+            </tr>
+            <tr>
+              <td>squeeze_left</td>
+             <td>Xbox Elite Wireless Controller<br>
+                PlayStation DualSense Edge</td>
+            </tr>
+            <tr>
+              <td>squeeze_left_lower</td>
+              <td>Xbox Elite Wireless Controller</td>
+            </tr>
+            <tr>
+              <td>squeeze_right</td>
+             <td>Xbox Elite Wireless Controller<br>
+                PlayStation DualSense Edge</td>
+            </tr>
+            <tr>
+              <td>squeeze_right_lower</td>
+              <td>Xbox Elite Wireless Controller</td>
+            </tr>
+          </tbody>
+        </table>
+       <section>
+          <h3>
+            <dfn>GamepadButtonType</dfn> Enum
```

Comment:
I see 8 different classes of extended buttons in the table above, but only 2 applicable GamepadButtonTypes in the enum definition. Is the intention to expand the enum list below so we have a 1:1 mapping with the rows in the table?

Reply:
updated.